tests/conftest.py

```diff
@@ -288,3 +305,129 @@ def patch_infer_revision(mocker):
     )
 
     yield
+
+
+@pytest.fixture
+def backend_client(garden_client):
```

Comment:
understanding check - what's the difference between the BackendClient returned by this fixture and the one at `garden_client.backend_client`?

Reply:
There isn't a difference functionally. I think it makes sense in `test_backend_client.py` to have a BackendClient to test directly rather than having to use `garden_client.backend_client....` in each test (even if that is essentially what happens in the fixture).